common_test.go

```diff
@@ -460,7 +458,7 @@ func (s *EnryTestSuite) TestGetLanguageGroup() {
 		expected string
 	}{
 		{name: "TestGetLanguageGroup_1", language: "BestLanguageEver", expected: ""},
-		{name: "TestGetLanguageGroup_2", language: "JSX", expected: "JavaScript"},
+		{name: "TestGetLanguageGroup_2", language: "Bison", expected: "Yacc"},
```

Comment:
I'm a bit confused about why is this change needed? Did the mock data we use changed elsewhere?

Reply:
JSX was removed as language and `.jsx` extension was just added to `JavaScript`

Reply:
Did not realise that - thank you so much for explaining!